ci-operator/step-registry/baremetalds/assisted/conf/devscripts/extraworkers/baremetalds-assisted-conf-devscripts-extraworkers-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: baremetalds-assisted-conf-devscripts-extraworkers
+  from: baremetal-installer
+  commands: baremetalds-assisted-conf-devscripts-extraworkers-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 500Mi
+  env:
+    - name: NUM_EXTRA_WORKERS
+      default: "0"
+      documentation: Indicate number of extra VMs to create but not deploy.
+    - name: APPLY_EXTRA_WORKERS
+      default: "false"
+      documentation: Apply extra workers automatically if NUM_EXTRA_WORKERS is set.
```

Comment:
I see that this variable isn't changing. Do we actually need it to be configurable?

Reply:
Probably not, and it seems to be apply even when set like this. This one might be a straight boolean.